app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -93,7 +92,7 @@ fun Menu() {
             horizontalArrangement = Arrangement.Center,
             verticalAlignment = Alignment.CenterVertically
         ) {
-            Text(text= countryName, fontSize = 18.sp, modifier = Modifier.padding(end = 8.dp))
+            Text(text= menuName, fontSize = 18.sp, modifier = Modifier.padding(end = 8.dp))
```

Comment:
changed this variable name as it makes more sense with the theme of the project

Reply:
Good.  Variable names should be descriptive.